scripts/deploy-sites-docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker exec -it pointnetwork_website_owner_1 /app/scripts/deploy-sites.sh /data
```

Comment:
`docker exec -it website_owner /app/scripts/deploy-sites.sh /data`

Reply:
@Tunji17 thanks for the notice. The service name in the docker-compose runtime will be prefixed with `pointnetwork_` and extended with an index, so it will look like `pointnetwork_website_owner_1`. I'm not sure the command will work with bare service name. I'll check.